frontend/packages/ceph-storage-plugin/src/queries/ceph-queries.ts

```diff
@@ -233,6 +237,15 @@ export const getPoolQuery = (poolNames: string[], queryName: string) => {
     [StorageDashboardQuery.POOL_UTILIZATION_THROUGHPUT_QUERY]: `(rate(ceph_pool_wr_bytes[1m]) + rate(ceph_pool_rd_bytes[1m])) * on (pool_id) group_left(name)ceph_pool_metadata{name=~'${poolNames.join(
       '|',
     )}'}`,
+    [StorageDashboardQuery.POOL_COMPRESSION_SAVINGS]: `(ceph_pool_compress_under_bytes - ceph_pool_compress_bytes_used) * on (pool_id) group_left(name)ceph_pool_metadata{name=~'${poolNames.join(
+      '|',
+    )}'}`,
+    [StorageDashboardQuery.POOL_COMPRESSION_ELIGIBILITY]: `((sum(ceph_pool_compress_under_bytes > 0) / sum(ceph_pool_stored_raw and ceph_pool_compress_under_bytes > 0)) * 100) * on (pool_id) group_left(name)ceph_pool_metadata{name=~'${poolNames.join(
+      '|',
+    )}'}`,
+    [StorageDashboardQuery.POOL_COMPRESSION_RATIO]: `((ceph_pool_compress_under_bytes / ceph_pool_compress_bytes_used > 0) and on(pool_id) (((ceph_pool_compress_under_bytes > 0) / ceph_pool_stored_raw) * 100 > 0.5)) * on (pool_id) group_left(name)ceph_pool_metadata{name=~'${poolNames.join(
+      '|',
```

Comment:
I am wondering these queries are correct, Please check with @anmolsachan  about these queries

Reply:
Checked with @anmolsachan . Queries generally look good. But as per @anmolsachan , some general changes (which is general for all the querries, not just for compression querries), we may have to add prometheus recording rules, but on a later stage.

Reply:
ack